src/lib.rs

```diff
@@ -136,22 +136,31 @@ pub type Delay = std::time::Duration;
 /// If the packet 101 is not lost, then the probability of packet 102 being lost is still 0.1.
 pub type LossPattern = Vec<f64>;
 
-/// Similar to loss_pattern, the duplicate_pattern describes how the packets are duplicated when going through.
+/// The duplicate_pattern describes how the packets are duplicated when going through the device.
 ///
-/// The duplicate_pattern is a sequence of conditional probabilities describing how packets are duplicated.
+/// The duplicate_pattern is a sequence of conditional probabilities describing how packets are
+/// duplicated.
 /// The probability is a f64 between 0 and 1.
 ///
 /// The meaning of the duplicate_pattern sequence is:
 ///
-/// - The probability on index 0 describes how likely a packet will be duplicated **if the previous packet was transmitted normally**.
-/// - The probability on index 1 describes how likely a packet will be duplicated **if the previous packet was duplicated**.
+/// - The probability on index 0 describes how likely a packet will be duplicated **if the previous
+/// packet was transmitted normally**.
+/// - The probability on index 1 describes how likely a packet will be duplicated **if the previous
+/// packet was
+/// duplicated**.
```

Comment:
These lines are weird. Please make some adjustments. Let all words in ** in one line.